llvm/lib/Transforms/Vectorize/LoopVectorizationLegality.cpp

```diff
@@ -78,6 +78,11 @@ static cl::opt<LoopVectorizeHints::ScalableForceKind>
                 "Scalable vectorization is available and favored when the "
                 "cost is inconclusive.")));
 
+static cl::opt<bool> UseWiderVFIfVariantsPresent(
+    "vectorizer-maximize-bandwidth-if-variant-present", cl::init(true),
```

Comment:
Again, it's not obvious this is a variant of what? How about something like

maximize-vector-bandwidth-for-vector-calls

?